src/Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -122,12 +123,14 @@ public void MergeInto(TextFileConfiguration destination)
         /// </summary>
         /// <param name="key">Name of the argument (excluding the dash prefix).</param>
         /// <returns>Values for the specified argument.</returns>
-        public string[] GetAll(string key)
+        public string[] GetAll(string key, ILogger logger = null)
```

Comment:
Please add param name xmldoc for the logger

Reply:
Done.